impeller/renderer/backend/vulkan/command_encoder_vk.cc

```diff
@@ -222,12 +223,13 @@ fml::StatusOr<vk::DescriptorSet> CommandEncoderVK::AllocateDescriptorSets(
                                                                       context);
 }
 
-void CommandEncoderVK::PushDebugGroup(const char* label) const {
+void CommandEncoderVK::PushDebugGroup(std::string_view label) const {
   if (!HasValidationLayers()) {
```

Comment:
This is basically the improvement, since the labels are invisible if there are no validation errors, moving the allocation below means most of the time (benchmarks) they wont even be allocated.